src/test/polymer-bundler_test.ts

```diff
@@ -46,6 +47,30 @@ suite('polymer-bundler CLI', () => {
     assert.include(stdout, 'hello from /absolute-paths/script.js');
   });
 
+  suite('--manifest-out', () => {
+
+    test('writes out the bundle manifest to given path', async () => {
+      const projectRoot = path.resolve(__dirname, '../../test/html');
+      const tempdir = tempy.directory();
```

Comment:
Can you use https://nodejs.org/api/fs.html#fs_fs_mkdtempsync_prefix_options instead?